internal/parser.go

```diff
@@ -43,11 +43,10 @@ type (
 	taskList map[string]Task
 )
 
-const osCommandRegexp = `\$\((.+)\)`
-
+var osCommandRegexp = regexp.MustCompile(`\$\((.+)\)`)
 var parserString string
 
-// Provide a parser instance which can be either a blank one,
+//NewParser Provide a parser instance which can be either a blank one,
```

Comment:
What's the reason to add the method name within the comment? Also, there should be a space after `//`

Reply:
The method name at the beginning is common across godoc :
https://go.dev/blog/godoc

Example from path.Join:
`// Join joins any number of path elements into a single path,
// separating them with slashes. Empty elements are ignored.
// The result is Cleaned. However, if the argument list is
// empty or all its elements are empty, Join returns
// an empty string.
func Join(elem ...string) string {`

As for the space:
there is no formal convention, jetbrains auto formats it to no-space, which is what i did, but i can leave the space (as the go core funcs do it)

Reply:
Thanks for explaining!